script/Ciel/Day 11/11_01_CIEL11_2_1.txt

```diff
@@ -238,7 +254,7 @@ Watching this scene unfold fills my heart with excitement, but I'll take my leav
 [sha:ec59d7ec15ccaf6fc68beca76a39f2ed92e53bcf]{
 -- Page 111, Offset 29373.
 -- 自分自身に食事の<献立|メニュー>やら<作法|マナー>やらを評価されるのは興醒めだ。
-Having the menu and manner of my own feeding evaluated would kill the mood.
+Having my own self critique the menu and my table manners would only spoil the mood.
```

Comment:
This sentence feels a bit odd in how it goes between the different subjects.

>Having my menu and table manners critiqued by my very own self would spoil the mood.

This sounds better to me.